### PR TITLE
Feature: security module

### PR DESCRIPTION
**Pull request by lciolecki**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -561,20 +562,23 @@ lazy val api = (project in engine("api")).
...
   settings(commonSettings).
   settings(
-    name := "nussknacker-security-api",
+    name := "nussknacker-security",
     libraryDependencies ++= {
       Seq(
         "org.scalatest" %% "scalatest" % scalaTestV % "test",
+        "org.mindrot" % "jbcrypt" % jbcryptV,
         "com.typesafe.akka" %% "akka-http" % akkaHttpV force(),
+        "com.typesafe.akka" %% "akka-stream" % akkaV force(),
```
Comment (arkadius) [inline build.sbt]: Can all this stuff provided to be sure that in final security extension pack, wouldn't be those libs? Will it work with transitive dependencies?

File: build.sbt [hunk 2]
```diff
@@ -561,20 +562,23 @@ lazy val api = (project in engine("api")).
...
+        "de.heikoseeberger" %% "akka-http-circe" % akkaHttpCirceV,
         "com.typesafe.akka" %% "akka-http-testkit" % akkaHttpV % "test" force(),
         "com.typesafe" % "config" % configV
       )
     }
   )
-  .dependsOn(util)
+  .dependsOn(util, httpUtils)
```
Comment (arkadius) [inline build.sbt]: The same as above

Comment (lciolecki) [reply build.sbt]: Changed.

File: build.sbt (hunk 1 above)
Comment (lciolecki) [reply build.sbt]: Good idea! Changed.

**Pull request merged**